examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -582,10 +591,14 @@ static sl_status_t wfx_rsi_do_join(void)
 
         sl_wifi_set_join_callback(join_callback_handler, NULL);
 
+#if SL_ICD_ENABLED
         // Setting the listen interval to 0 which will set it to DTIM interval
         sl_wifi_listen_interval_t sleep_interval = { .listen_interval = 0 };
         status                                   = sl_wifi_set_listen_interval(SL_WIFI_CLIENT_INTERFACE, sleep_interval);
 
+        sl_wifi_advanced_client_configuration_t client_config = { .max_retry_attempts = 5 };
```

Comment:
Same vein of my last comment. Other members are left to an undefined value. How does this impact the use of this variable. Any risk

Reply:
Based on C standard (6.7.9 )this is ok and the rest will be implicitly init

If there are fewer initializers in a brace-enclosed list than there are elements or members
of an aggregate, or fewer characters in a string literal used to initialize an array of known
size than there are elements in the array, the remainder of the aggregate shall be
initialized implicitly the same as objects that have static storage duration.